sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Bugs Fixed
 
+- Change the default value for the authority host option to be read from the environment variable first.
```

Comment:
I agree with Anton. The wording above makes it sound like a new feature, NOT a bug fix.

Arguably it's either of the two. But if you want to call it a bug fix, you should describe the change in terms of being a bug fix.

Something like:

> Fixed the order of evaluating the default authority host to match the behavior of other languages. Now the default value is read from an environment variable before <something>

Reply:
I can massage the changelog wording separately to avoid resetting CI, and waiting.

That said, this is a bug fix because the precedence order used is corrected.